doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -321,6 +317,37 @@ following steps:
    $ make clean && make iasl
    $ sudo cp ./generate/unix/bin/iasl /usr/sbin/
 
+Build and Install the RT kernel for the Ubuntu User VM
+------------------------------------------------------
+
+Follow these instructions to build the RT kernel.
+
+#. Clone the RT kernel source code:
+
+   .. code-block:: none
+
+      $ git clone https://github.com/projectacrn/acrn-kernel/tree/4.19/preempt-rt
+
+#. Build the kernel:
+
+   .. code-block:: none
+
+      $make olddefconfig
+      $make targz-pkg
+
+#. Copy the kernel and modules:
+
+   .. code-block:: none
+
+      $ mount /dev/sda1 /mnt
+      $ mkdir /mnt/EFI/acrn
```

Comment:
It's unusual to put the kernel (`bzImage`) in such folder, we usually only put the `acrn.efi` hypervisor in there. While it's OK to do so, I'd rather we do not. And it's also inconsistent with the `/etc/grub.d/40_custom` we use below.... so my recommendation is to skip this `mkdir` step and just copy `bzImage` to `/mnt/EFI/`

Reply:
Updated to:
      $ sudo mount /dev/sda1 /mnt
      $ sudo cp bzImage /mnt/EFI/
      $ sudo umount /mnt

...snip...